testsuite/Dockerfile

```diff
@@ -0,0 +1,32 @@
+# Use default azure-cli version 2.29.0 if no version is specified in build
+ARG AZURE_CLI_VERSION=2.29.0
```

Comment:
Let's move this default value to pipeline itself instead of declaring here. We have done the similar kind of thing here: 
https://github.com/Azure/azure-arc-kubernetes-conformance/blob/9068583e5933183849504f00869eb53c28d4f1e4/src/plugins/arc_agent_cleanup/Dockerfile#L12

Reply:
ack